recipes/llama-index-embeddings-azure-openai/recipe.yaml

```diff
@@ -0,0 +1,46 @@
+context:
+  version: 0.3.0
+  python_min: "3.9"
```

Comment:
I'm not 100% sure for the new recipe format, but in theory you don't need to set this is they are the same as conda-forge min. You only need this if the package min is higher than conda-forge's.

```suggestion
```

Reply:
oops yes this was for local testing. will fix.